src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -26,16 +26,29 @@ object WdlExpression {
 
   val unaryOperators = Set("LogicalNot", "UnaryPlus", "UnaryNegation")
 
-  def evaluate(ast: AstNode, lookup: ScopedLookupFunction, functions: WdlFunctions): Try[WdlValue] = {
+  def interpolate(str: String, lookup: ScopedLookupFunction): String = {
+    val regex = "\\$\\{([a-z]+)\\}".r
+    var interpolated = str
```

Comment:
An example of how it might work:

    def interpolateTag(interpolated: String, tag: String) = {
      val variable = tag.substring(2, tag.length - 1)
      val replacement = lookup(variable)
      interpolated.replace(tag, replacement.valueString)
    }
    tags.foldLeft("")(interpolateTag)

Note that I didn't try this as-is but I set up a simplistic example that was similar and it worked AOK:

    val tags = Vector("#", "@", "*")
    val foo = "asdf#etet@kjkj*"
    def interp(target: String, tag: String) = target.replace(tag, " ")
    tags.foldLeft(foo)(interp)

    scala> tags.foldLeft(foo)(interp)
    res8: String = "asdf etet kjkj "